components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -46,28 +47,41 @@ void SurfaceCouplingExporter::set_grids(const std::shared_ptr<const GridsManager
   constexpr int ps = Spack::n;
 
   // These fields are required for computation/exports
-  add_field<Required>("p_int",                scalar3d_layout_int,  Pa,    grid_name);
-  add_field<Required>("pseudo_density",       scalar3d_layout_mid,  Pa,    grid_name, ps);
-  add_field<Required>("phis",                 scalar2d_layout,      m2/s2, grid_name);
-  add_field<Required>("p_mid",                scalar3d_layout_mid,  Pa,    grid_name, ps);
-  add_field<Required>("qv",                   scalar3d_layout_mid,  Qunit, grid_name, "tracers", ps);
-  add_field<Required>("T_mid",                scalar3d_layout_mid,  K,     grid_name, ps);
-  add_field<Required>("horiz_winds",          vector3d_layout,      m/s,   grid_name);
-  add_field<Required>("sfc_flux_dir_nir",     scalar2d_layout,      Wm2,   grid_name);
-  add_field<Required>("sfc_flux_dir_vis",     scalar2d_layout,      Wm2,   grid_name);
-  add_field<Required>("sfc_flux_dif_nir",     scalar2d_layout,      Wm2,   grid_name);
-  add_field<Required>("sfc_flux_dif_vis",     scalar2d_layout,      Wm2,   grid_name);
-  add_field<Required>("sfc_flux_sw_net" ,     scalar2d_layout,      Wm2,   grid_name);
-  add_field<Required>("sfc_flux_lw_dn"  ,     scalar2d_layout,      Wm2,   grid_name);
+  add_field<Required>("p_int",                scalar3d_layout_int,  Pa,     grid_name);
+  add_field<Required>("pseudo_density",       scalar3d_layout_mid,  Pa,     grid_name, ps);
+  add_field<Required>("phis",                 scalar2d_layout,      m2/s2,  grid_name);
+  add_field<Required>("p_mid",                scalar3d_layout_mid,  Pa,     grid_name, ps);
+  add_field<Required>("qv",                   scalar3d_layout_mid,  Qunit,  grid_name, "tracers", ps);
+  add_field<Required>("T_mid",                scalar3d_layout_mid,  K,      grid_name, ps);
+  add_field<Required>("horiz_winds",          vector3d_layout,      m/s,    grid_name);
+//ASD  add_field<Required>("u",                    scalar3d_layout_mid,  m/s,    grid_name); // TODO: These appear to not work yet
```

Comment:
Yeah, the u/v fields are created by the AD _after_ all fields have been created. If you declare them here, the AD will think that they are unrelated to `horiz_winds`. I don't have a good way to fix this yet.